submission/submission-core/src/main/kotlin/ac/uk/ebi/biostd/submission/domain/submitter/ExtSubmissionSubmitter.kt

```diff
@@ -5,11 +5,11 @@ import ebi.ac.uk.extended.model.ExtSubmission
 
 interface ExtSubmissionSubmitter {
     suspend fun createRequest(rqt: ExtSubmitRequest): Pair<String, Int>
-    suspend fun indexRequest(accNo: String, version: Int): Unit
-    suspend fun loadRequest(accNo: String, version: Int): Unit
-    suspend fun cleanRequest(accNo: String, version: Int): Unit
-    suspend fun processRequest(accNo: String, version: Int): Unit
-    suspend fun checkReleased(accNo: String, version: Int): Unit
+    suspend fun indexRequest(accNo: String, version: Int)
```

Comment:
i start adding explicit return type only to public api/functions as for example if you call a method in the last line of function body the return value becomes something and may break unit test for example.